src/main/resources/xslt/mods2solr.xsl

```diff
@@ -1,666 +1,555 @@
+            <xsl:value-of select="premis:size"/>
           </f:string>
         </xsl:if>
-        <f:string key="url">-->
-        <f:string>
-	  <xsl:choose>
-	    <xsl:when test="contains($img,'.json')">
-	      <xsl:value-of select="$img"/>
-	    </xsl:when>
-	    <xsl:otherwise>
-	      <xsl:value-of select="concat('https://kb-images.kb.dk/',$img,'/info.json')"/>
-	    </xsl:otherwise>
-	  </xsl:choose>            
+        <!-- Extracts the height of the digital image-->
+        <xsl:if test="premis:objectCharacteristicsExtension/mix:mix/mix:BasicImageInformation/mix:BasicImageCharacteristics/mix:imageHeight">
+        <f:string key="image_height">
+          <xsl:value-of select="premis:objectCharacteristicsExtension/mix:mix/mix:BasicImageInformation/mix:BasicImageCharacteristics/mix:imageHeight"/>
         </f:string>
-<!--      </f:map>-->
-    </xsl:if>
-  </xsl:template>
-
-  <xsl:template name="get-names">
-    <xsl:param name="record_identifier"/>
-    <f:map>
-      <xsl:call-template name="disposable-subrecord">
-        <xsl:with-param name="record_identifier" select="$record_identifier"/>
-      </xsl:call-template>
-      <xsl:if test="@authorityURI">
-        <f:string key="authority"><xsl:value-of select="@authorityURI"/></f:string>
-      </xsl:if>
-      <f:boolean key="described">false</f:boolean>
-      <f:string key="describing"><xsl:value-of select="$record_identifier"/></f:string>
-      <xsl:if test="@xml:lang">
-        <f:string key="language"><xsl:value-of select="@xml:lang"/></f:string>
-      </xsl:if>
-      <f:string key="entity_type"><xsl:choose>
-                      <xsl:when test="contains(m:role/m:roleTerm,'src')">scr</xsl:when>
-                      <xsl:otherwise><xsl:value-of select="m:role/m:roleTerm"/></xsl:otherwise>
-                    </xsl:choose></f:string>
-      <f:string key="agent_name">
-        <xsl:for-each select="m:namePart">
-          <xsl:choose>
-            <xsl:when test="@type = 'date'"> (<xsl:value-of select="."/>)</xsl:when>
-            <xsl:otherwise><xsl:value-of select="."/></xsl:otherwise>
-          </xsl:choose>
-        </xsl:for-each>
-      </f:string>
+        </xsl:if>
+        <!-- Extracts the width of the digital image-->
+        <xsl:if test="premis:objectCharacteristicsExtension/mix:mix/mix:BasicImageInformation/mix:BasicImageCharacteristics/mix:imageWidth">
+        <f:string key="image_width">
+          <xsl:value-of select="premis:objectCharacteristicsExtension/mix:mix/mix:BasicImageInformation/mix:BasicImageCharacteristics/mix:imageWidth"/>
+        </f:string>
+        </xsl:if>
+        <!-- Calculates the size of the digital image-->
+        <xsl:if test="premis:objectCharacteristicsExtension/mix:mix/mix:BasicImageInformation/mix:BasicImageCharacteristics/mix:imageHeight * premis:objectCharacteristicsExtension/mix:mix/mix:BasicImageInformation/mix:BasicImageCharacteristics/mix:imageWidth">
+        <f:string key="image_size">
```

Comment:
`image_size` is ambiguous. Could we use `image_size_pixels` or `image_pixels` instead?

Also: `image_size` is defined as a `float` in `schema.xml`!? It should be a `long`.

Reply:
Nice catch with the schema format also. Missed it.
Victor, you fix this? About the field name, maybe there is a reason ?

Reply:
changed to image_size_pixels